queries/heex/highlights.scm

```diff
@@ -1,16 +1,34 @@
-(text) @text
-(comment) @comment
 (doctype) @constant
 
+(text) @text
+
+; Phoenix bindings
+((attribute_name) @keyword (#match? @keyword "^phx-"))
```

Comment:
Do you have to escape to mean  a verbatim `-` -> `%-`

Reply:
Thanks for the feedback! I decided to remove this highlight since it is not really part of HEEx itself, but how Phoenix Live View interprets it.